core/src/main/scala/akka/kafka/ConsumerSettings.scala

```diff
@@ -71,6 +71,12 @@ object ConsumerSettings {
     val offsetForTimesTimeout = config.getDuration("offset-for-times-timeout").asScala
     val metadataRequestTimeout = config.getDuration("metadata-request-timeout").asScala
     val drainingCheckInterval = config.getDuration("eos-draining-check-interval").asScala
+
+    val enablePath = s"${KafkaConnectionCheckerSettings.configPath}.enable"
+    val kafkaConnectionCheckerConfig =
+      if (config.hasPath(enablePath) && config.getBoolean(enablePath)) Some(KafkaConnectionCheckerSettings(config))
```

Comment:
You need to use `config.getConfig(KafkaConnectionCheckerSettings.configPath)`, right?